inventory.yml.sample

```diff
@@ -122,8 +122,13 @@ all:
       }}"
     mirror_certificate: "{{ lookup('file', local_mirror_certificate_path) }}"
 
-    openshift_version: "{{ openshift_full_version.split('.')[:2] | join('.') }}"
+    openshift_full_version_mapping:
+      '4.6': 4.6.16
+      '4.7': 4.7.33
+      '4.8': 4.8.14
+      '4.9': 4.9.0
```

Comment:
I'm not sure if the mapping should be defined in the inventory. 

On the one hand, having it in the inventory makes it easier for the user to guess correctly what `openshift_version` will be expanded to.

On the other hand, it requires the user to manually update the inventory to have the mapping in line with `supported_ocp_versions` from `validate_inventory` after every new release is made. If the user still has to manually update the versions (just in a different place), what's the benefit?

Would it be possible to have the mapping defined somewhere in the roles' defaults instead? Are there any drawbacks to doing that?

Reply:
Good spot. I forgot to revert this change I was playing around with moving defaults like this and `supported_ocp_versions` into a separate inventory. I'll move it into role defaults.